docs/intro/index.mdx

```diff
@@ -34,27 +34,30 @@ testnet.
 - **Modular data storage**: Hybrid **onchain/offchain memory** built for AI workflows.
 
 Recall unifies these aspects by providing a **structured, machine-verifiable intelligence layer**
-for AI and crypto ecosystems. With Recall’s [agent framework integrations](/agent/plugins),
+for AI and crypto ecosystems. With Recall's [agent framework integrations](/agent/plugins),
 [memory blockchain](/fundamentals), and intelligence portal (coming soon), developers gain an
 **unprecedented ability to test, refine, and showcase their agents** while enabling trustless data
 exchange and monetization.
 
 ```mermaid
-graph LR
-    subgraph "Decision layer for AI"
-        %% Core Recall Node
-        Agents --> R[Recall]
-        %% Capabilities
-        R -->|Verify| V[✓]
-        R -->|Trade| T[$]
-        %% Agent Improvement Loop
-        R -->|Thoughts| S[🧠]
-        R -->|Memories| M[💭]
-        R -->|Knowledge| C[📈]
-        %% Improvement Cycle
-        S & M & C --> L[Improve]
-        L --"Observe"--> Agents
-    end
+graph TD
```

Comment:
note: snagged this from [here](https://www.notion.so/mermaid-noodling-19edfc9427de8044966cee641272aa8c?pvs=4)